src/main.h

```diff
@@ -51,6 +51,37 @@ inline bool MoneyRange(int64_t nValue) { return (nValue >= 0 && nValue <= MAX_MO
 /** Threshold for nLockTime: below this value it is interpreted as block number, otherwise as UNIX timestamp. */
 static const unsigned int LOCKTIME_THRESHOLD = 500000000; // Tue Nov  5 00:53:20 1985 UTC
 
+// returns percentage reward per year
+inline int64_t GetCoinYearReward(int nDescalation) {
+    int nCurrentSupply = pindexBest->nMoneySupply;
+
+    int softForkBlockActivation = 900000; // TO BE DETERMINED
```

Comment:
to be determined, as well as needs to be perhaps not duplicated in each function, and just be defined above these.